packages/next/package.json

```diff
@@ -234,7 +234,10 @@
     "async-retry": "1.2.3",
     "async-sema": "3.0.0",
     "axe-playwright": "2.0.3",
+    "babel-loader": "10.0.0",
     "babel-plugin-react-compiler": "19.1.0-rc.2",
+    "babel-plugin-syntax-jsx": "6.18.0",
```

Comment:
why we use a old version of syntax-jsx not `@babel/plugin-syntax-jsx` v7? and babel-plugin-syntax-typescript is v7 alpha?

Reply:
Accidentally. Now using `@babel/plugin-syntax-typescript`.